src/modules/commander/PreflightCheck.cpp

```diff
@@ -262,7 +262,7 @@ static bool accelerometerCheck(orb_advert_t *mavlink_log_pub, unsigned instance,
 
 	if (ret) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCEL #%u UNCALIBRATED", instance);
+			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCELEROMETER #%u UNCALIBRATED.", instance);
```

Comment:
Capitalization already improved in this more recent PR: https://github.com/PX4/Firmware/pull/10586/files#diff-a2bcb238e25d8a97ab2aee48f54dbc48R262
However, using full word "accelerometer" is probably better than "Accel"
Suggested new text: "Preflight Fail: accelerometer #%u uncalibrated."